openfisca_core/scripts/measure_numpy_condition_notations.py

```diff
@@ -10,6 +10,8 @@
 
 The aim of this script is to compare the time taken by the calculation of the values
 """
+from __future__ import print_function
```

Comment:
What features is `print_function`bringing in this file ?

Reply:
Could we write :
```py
from __future__  import unicode_literals, print_function
```

Reply:
Not sure, [`futurize`](http://python-future.org/futurize.html) added that for me 😄 (I know, bad answer).

I don't think it really brings features per se. It just provides the module the Python 3 version of print. The only concrete advantages I can see:
-  if a contributor on Python 2 uses `print` the old way, they will get the error early, instead of waiting for the CI to crash.
- Running futurize again will not suggest the same change again

Reply:
Actually, I think to avoid confusion and to avoid inconsistency, let me just import all the __future__ in every module, on one line.